pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["CA"]
 homepage = "https://github.com/citizensadvice/ca-cdk-constructs"
 
 [tool.poetry.dependencies]
-python = ">=3.11 <3.12"
+python = ">=3.10 <3.11"
```

Comment:
missing comma?

Reply:
also I think you might need explicit `poetry lock` + commit  + push to fix the exceptiongroup error

Reply:
You're right, my mistake